python2/koans/about_scoring_project.py

```diff
@@ -34,8 +34,21 @@
 # Your goal is to write the score method.
 
 def score(dice):
+    final_score = 0
+    repeated_nums = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+    for number in dice:
+        repeated_nums[number]=repeated_nums[number] + 1
+    for i in range(0,10):
+        aux1 = int (repeated_nums[i] / 3)
+        aux2 = repeated_nums[i] % 3
+        if i == 1:
+            final_score = final_score + (aux1 * 1000) + (aux2 * 100)
+        elif i == 5:
+            final_score = final_score + (aux1 * 100 * 5) + (aux2 * 50)
+        else:
+            final_score = final_score + (aux1 * 100 * i)
```

Comment:
Changing the length of `repeated_nums` back to 6 will require some changes to this section of the code, as `i` will only go from 0 to 5. In this set of `if`s, we want `i` to represent the die value, and not the list index. To do so, we can just add 1 to `i`, to revert the `i-1` we had to apply to adjust the dice values to the list indexes.

```python
# This change the values of i from 0-5 to 1-6.
i += 1
if i == 1:
    final_score = final_score + (aux1 * 1000) + (aux2 * 100)
elif i == 5:
    final_score = final_score + (aux1 * 100 * 5) + (aux2 * 50)
else:
    final_score = final_score + (aux1 * 100 * i)
```